sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/common/ChangeFeedProcessorItem.java

```diff
@@ -15,7 +14,6 @@
  *
  * Caller is recommended to type cast {@link JsonNode} to cosmos item structure.
  */
-@Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public final class ChangeFeedProcessorItem {
```

Comment:
same here, please revert this.

Reply:
Implementation package - no Beta needed.